node/tests/SharedTests.ts

```diff
@@ -2775,6 +2775,68 @@ export function runBaseTests<Context>(config: {
         config.timeout,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        `expiretime and pexpiretime test_%p`,
+        async (protocol) => {
+            await runTest(async (client: BaseClient, cluster) => {
+                if (cluster.checkIfServerVersionLessThan("7.0.0")) {
+                    return;
+                }
+
+                const key = uuidv4();
+
+                expect(await client.set(key, "foo")).toEqual("OK");
+                expect(await client.ttl(key)).toEqual(-1);
```

Comment:
Is this relevant? Based on Java and Python, do we just need to add to an existing test?

Reply:
i was following existing python and java tests for this shared test

Reply:
Right, can you add lines to this test instead? `expire, pexpire, expireAt and pexpireAt with timestamp in the past or negative timeout`

That way we don't have to duplicate the test.